resources/eventing/charts/controller/templates/deployment.yaml

```diff
@@ -132,7 +132,7 @@ spec:
         - name: cert
           secret:
             defaultMode: 420
-            secretName: eventing-webhook-server-cert
+            secretName: {{ .Values.webhook.secretName }}
```

Comment:
Let's change this [here](https://github.com/kyma-project/kyma/blob/93184b838bf714cf6965a185cf27be7615706b99/resources/eventing/charts/controller/templates/deployment.yaml#L135) as well!

Reply:
And delete this file: 
https://github.com/kyma-project/kyma/blob/main/resources/eventing/charts/controller/templates/certmanager.yaml

We need to delete [this](https://github.com/kyma-project/kyma/tree/main/components/eventing-controller/config/certmanager) as well.